frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { untilNoLoadersPresent } from '@console/internal-integration-tests/views/crud.view';
+import {
+  obcCount,
+  efficiencyValue,
+  savingsValue,
+  healthOfMCG,
+  resourceProviders,
+  noobaaCount,
+} from '../views/noobaaDashboardPage.view';
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: testName,
+    name: 'test-bucket',
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Tests at least one noobaa bucket is present', async () => {
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count, ,] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Creates an Object Bucket Claim and tests equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -n ${testName} -f -`);
+    await browser.sleep(10000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -n ${testName} -f -`);
+  });
+});
+
+describe('Tests Status Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks if Multi Cloud Gateway is in a healthy state', async () => {
```

Comment:
I don't find this valid. The integration test should say the state of multi-cloud gateway is reflected properly. We should get the state from noobaa command and check this. Noobaa can come up in an unhealthy state and in that case, this test will fail.

Reply:
we cannot assume that the noobaa command line is available.

Reply:
What possible things are there that can cause Noobaa to go unhealthy? If it can go unhealthy due to some tests then we should run this before that. But in an ideal deployment of Noobaa, I am assuming that Noobaa would come up healthy. Is this a safe assumption? If not why cannot Noobaa come up healthy?